### PR TITLE
Install & Configure Wordpress

### PR DESCRIPTION
**Pull request by JuniYadi**
This tutorial is a connection from my previous tutorial that has been published [HERE](https://community.hetzner.com/tutorials/install-lamp-server-on-centos-7).

### CODE REVIEW

**Comment by JuniYadi** [comment]
> Can you add the new language tags from the [Template File](https://github.com/hetzneronline/community-content/blob/master/tutorial-template.md), please?
> 
> `language: "en"`
> `available_languages: ["en"]`

Done

File: tutorials/install-and-configure-wordpress-on-centos-7/01.en.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+cp -r * /var/www/html/example.com/public_html
+chown -R apache:apache /var/www/html/example.com/public_html
+find /var/www/html/example.com/public_html -type d -exec chmod 755 {} \;
+find /var/www/html/example.com/public_html -type f -exec chmod 644 {} \;
+
+```
+
+* For holu.example.com:
+```bash
+cp -r * /var/www/html/holu.example.com/public_html
```
Comment (n-se) [inline tutorials/install-and-configure-wordpress-on-centos-7/01.en.md]: First of all, when did you created /var/www/html/holu.example.com(/public_html)? You cannot copy with `-r` if this folder does not exist. Second, why do we need a public_html folder inside our domain folder?

As a side note, why do you put other websites into `/html`? Is this how it is done on CentOS?
I mean normally you do it like `/var/www/example.com` and `/var/www/holu.example.com` and a new vhost for each of them?

Comment (JuniYadi) [reply tutorials/install-and-configure-wordpress-on-centos-7/01.en.md]: all path it's using same version with [LAMP Stack](https://community.hetzner.com/tutorials/install-lamp-server-on-centos-7) tutorial before, I have written this on: Prerequisites.

that's why in this tutorial there is no vhost setting for example.com and holu.example.com.

public_html is needed only to save the main website file, while for the domain folder there will be 2 other files namely error.log and requests.log, this stores all error data and requests for each website, and if you want a private folder, you can create it outside the public_html folder.

`
[root@your_host example.com]# ls
error.log  public_html  requests.log

[root@your_host example.com]# tail -10 error.log
[Mon Mar 25 11:48:06.643461 2019] [php7:error] [pid 24301] [client xxx.xxx.xxx.xxx:21028] script '/var/www/html/example.com/public_html/aotu7.php' not found or unable to stat
[Mon Mar 25 11:48:06.997838 2019] [php7:error] [pid 24301] [client xxx.xxx.xxx.xxx:21028] script '/var/www/html/example.com/public_html/cmd.php' not found or unable to stat
[Mon Mar 25 11:48:10.144176 2019] [php7:error] [pid 24301] [client xxx.xxx.xxx.xxx:21028] script '/var/www/html/example.com/public_html/bak.php' not found or unable to stat
[Mon Mar 25 11:48:10.613709 2019] [php7:error] [pid 24301] [client xxx.xxx.xxx.xxx:21028] script '/var/www/html/example.com/public_html/system.php' not found or unable to stat
[Mon Mar 25 11:48:13.177871 2019] [php7:error] [pid 24301] [client xxx.xxx.xxx.xxx:21028] script '/var/www/html/example.com/public_html/l6.php' not found or unable to stat
[Mon Mar 25 11:48:14.117553 2019] [php7:error] [pid 24301] [client xxx.xxx.xxx.xxx:21028] script '/var/www/html/example.com/public_html/l7.php' not found or unable to stat
[Mon Mar 25 11:48:26.147353 2019] [php7:error] [pid 24196] [client xxx.xxx.xxx.xxx:23750] script '/var/www/html/example.com/public_html/q.php' not found or unable to stat
[Mon Mar 25 11:48:26.463559 2019] [php7:error] [pid 24196] [client xxx.xxx.xxx.xxx:23750] script '/var/www/html/example.com/public_html/56.php' not found or unable to stat
[Mon Mar 25 11:48:28.948089 2019] [php7:error] [pid 24196] [client xxx.xxx.xxx.xxx:23750] script '/var/www/html/example.com/public_html/mz.php' not found or unable to stat
[Mon Mar 25 11:48:29.291692 2019] [php7:error] [pid 24196] [client xxx.xxx.xxx.xxx:23750] script '/var/www/html/example.com/public_html/xx.php' not found or unable to stat

[root@your_host example.com]# tail -10 requests.log
94.102.49.193 - - [26/Mar/2019:21:16:33 -0400] "GET / HTTP/1.1" 200 11076 "-" "Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36"
94.102.49.193 - - [26/Mar/2019:21:16:37 -0400] "GET /robots.txt HTTP/1.1" 404 208 "-" "-"
94.102.49.193 - - [26/Mar/2019:21:16:40 -0400] "GET /sitemap.xml HTTP/1.1" 404 209 "-" "-"
94.102.49.193 - - [26/Mar/2019:21:16:43 -0400] "GET /.well-known/security.txt HTTP/1.1" 404 222 "-" "-"
94.102.49.193 - - [26/Mar/2019:21:16:53 -0400] "GET /favicon.ico HTTP/1.1" 404 209 "-" "python-requests/2.10.0"
51.79.128.119 - - [26/Mar/2019:22:49:30 -0400] "POST /wp-cron.php?doing_wp_cron=1553654970.0590291023254394531250 HTTP/1.1" 200 - "http://example.com/wp-cron.php?doing_wp_cron=1553654970.0590291023254394531250" "WordPress/5.1.1; http://example.com"
172.104.108.109 - - [26/Mar/2019:22:49:29 -0400] "GET / HTTP/1.1" 301 - "-" "Go-http-client/1.1"
172.104.108.109 - - [26/Mar/2019:22:49:30 -0400] "GET / HTTP/1.1" 200 11076 "http://51.79.128.119:80" "Go-http-client/1.1"
185.64.208.81 - - [26/Mar/2019:23:58:45 -0400] "GET / HTTP/1.1" 301 - "-" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36"
51.79.128.119 - - [26/Mar/2019:23:58:46 -0400] "POST /wp-cron.php?doing_wp_cron=1553659126.1267468929290771484375 HTTP/1.1" 200 - "http://example.com/wp-cron.php?doing_wp_cron=1553659126.1267468929290771484375" "WordPress/5.1.1; http://example.com"
`

**Comment by JuniYadi** [comment]
> I am not completely sure, but isn't it better (or more correct) to use `sql` instead of `bash` on the mysql/mariadb code-blocks?

done

**Comment by n-se** [comment]
I have fixed some typos I have found and replaced the link to your previous tutorial with the correct one. Hope you are okay with these edits. Please revert any changes if not.

**Pull request merged**